### PR TITLE
py-solc: init at 3.2.0

### PR DESCRIPTION
**Pull request by paulperegud**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg build python3Packages.py-solc

**Comment by FRidh** [comment]
```
Ran 0 tests in 0.000s
```
No tests are found. Either the test runner cannot find tests and needs to be patched, or there simply are no tests in which case the tests need to be disabled. Do include a comment explaining why the tests are disabled.

**Comment by paulperegud** [comment]
@FRidh Disabling tests for now, since I can't get them running. Created an issue for that: https://github.com/ethereum/py-solc/issues/60

File: pkgs/development/python-modules/py-solc/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  propagatedBuildInputs = [ solc semantic-version ];
+
+  # checks are disabled because test suite seem to be broken:
+  # see here: https://github.com/ethereum/py-solc/issues/60
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python wrapper around the solc binary";
+    license = licenses.mit;
```
Comment (JohnAZoidberg) [inline pkgs/development/python-modules/py-solc/default.nix]: Please add `homepage = "https://github.com/ethereum/py-solc";`

**Comment by JohnAZoidberg** [comment]
Looks like they're not maintaining it anymore: https://github.com/ethereum/py-solc/pull/65#issuecomment-448729689
Doesn't even support the newest version of solc.

But he created a [fork](https://github.com/iamdefinitelyahuman/py-solc-x). Maybe you'd want to package that instead.

**Comment by paulperegud** [comment]
I will not need this one anymore.

**Pull request closed without merging**